### DISCUSSION TITLE
Quick question - tinymce

### DISCUSSION THREAD

**Discussion by Rodien** [2sic/2sxc#2939] [Q&A] [answered]
Hi guys,

Has anyone tried to add a custom a button to 2sxc tinymce.

Is it even possible?

**Comment by iJungleboy** [comment, marked as answer]
Yes, this is possible 😉

See https://2sxc.org/dnn-tutorials/en/razor/ui242/page

**Reply by Rodien** [reply]
Fantastic! i'm gonna take a look.

Thank you!

**Comment by Rodien** [comment]
Okay, so I think I'm missing something.

I added the following file just to test this out: wwwroot/Portals/3/2sxc/HTML/system/field-string-wysiwyg-button-ml/index.js

But nothing is happening.

**Reply by iJungleboy** [reply]
There is quite a bit more to it :) https://docs.2sxc.org/js-code/custom-fields/index.html

I assumed you had already done that, since you were only asking about a custom button, not about custom toolbars :D

**Reply by Rodien** [reply]
It's working now, thank you!🚀
